code/modules/mob/living/carbon/xenomorph/castes/hivemind/castedatum_hivemind.dm

```diff
@@ -90,21 +110,33 @@
 	// *** Evolution *** //
 	upgrade_threshold = 700
 
+	// *** Pheromones *** //
+	aura_strength = 3.3
+	aura_allowed = list("frenzy", "warding", "recovery")
+
 	// *** Abilities *** //
 	actions = list(
+		/datum/action/xeno_action/activable/reposition_core,
 		/datum/action/xeno_action/plant_weeds/slow,
 		/datum/action/xeno_action/psychic_whisper,
 		/datum/action/xeno_action/activable/psychic_cure,
 		/datum/action/xeno_action/toggle_pheromones,
+		// /datum/action/xeno_action/activable/mind_wrack, //Commenting this out for now until I submit a follow up PR that allows the Hivemind to manifest in a destructibe avatar.
```

Comment:
commented out code bad remove this or add it

Reply:
Fuck it, I'll give it an insane cooldown for now